src/main/java/org/checkerframework/specimin/FieldDeclarationsVisitor.java

```diff
@@ -30,6 +30,9 @@ public FieldDeclarationsVisitor() {
 
   @Override
   public void visit(FieldDeclaration decl, Void p) {
+    // Caution: Calling .get() on an Optional without checking if it's present.
+    // This will throw NoSuchElementException if the Optional returned by getParentNode()
+    // is empty.
```

Comment:
I don't find this comment very helpful - presumably this code was written based on an assumption that this call to `.get()` will never fail.

In this case, a `NoSuchElementException` can only occur here if there is a FieldDeclaration that does not have a parent node. I don't think that's possible in Java (at least not in Java 11, which is the language level that Specimin targets). So, the code is safe as-is. (A comment that explains why, like what I just wrote here, would be helpful.)

Reply:
I will remove comments similar to this one as well.